domains/upchat.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "chat website",
+  "owner": {
+    "username": "vk0900",
+    "email": "nnmvishnu@gmail.com"
+    "proxied": true
```

Comment:
Proxied does not go underneath the owner object, this is invalid. JSON is also invalid.